### ISSUE TITLE
[Symbols] showInIndexedReferenceList is undocumented

### ISSUE THREAD

**Issue by willstott101** [guillermooo/sublime-undocs#271]
The current reference/symbols docs makes no reference to the functionality demonstrated by the Default package's `Indexed Reference List.tmPreferences` file. It appears this file decides what comes under the References section of a Defintion/References popup for the indexed symbol list.

[The built-in Erlang package](https://github.com/sublimehq/Packages/blob/master/Erlang/Indexed%20Reference%20List.tmPreferences) seems to be the main user of the Symbol overrides and is the only one which includes `showInIndexedReferenceList`.

**Comment by FichteFoll** [comment]
That is correct. The reference list feature was added after that page has been written. I cannot reasonably apply changes to the docs currently, so this will be added to the backlog, but thanks for the heads-up.
